### DISCUSSION TITLE
Impact of grid size on Ta_canyon

### DISCUSSION THREAD

**Discussion by biglimp** [jixuan-chen/target#6] [General]
Sorry fro question bombing.

I tested to change the res-variable in the parameters file which I assume is the grid size. I see a small change in values keeping everything constant except for res. What meakes the values change? In my head they should be the same.

![image](https://github.com/user-attachments/assets/2b517c6e-5231-4feb-a8f0-c20aa1bedb54)

**Comment by jixuan-chen** [comment]
maybe because currently W is not taken from the LC-file but calculated in the code

**Reply by biglimp** [reply]
Ok, as I wrote in my email, I strongly recommend that W should not be calculated from fractions.

**Reply by jixuan-chen** [reply]
I think I got your point. I'll change it.
